packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -106,16 +106,25 @@ class CupertinoDialog extends StatelessWidget {
 /// An iOS-style alert dialog.
 ///
 /// An alert dialog informs the user about situations that require
-/// acknowledgement. An alert dialog has an optional title and an optional list
-/// of actions. The title is displayed above the content and the actions are
-/// displayed below the content.
+/// acknowledgement. An alert dialog has an optional title, optional content,
+/// and an optional list of actions. The title is displayed above the content
+/// and the actions are displayed below the content.
+///
+/// This dialog styles its title and content to match the standard iOS title
+/// and content dialog text style. These default styles can be overridden by
```

Comment:
iOS has a title and a message. We can just say `This dialog styles its title and content (typically a message) to match the standard iOS title and message dialog text style`